test/wsdl/recursive/A.xsd

```diff
@@ -0,0 +1,25 @@
+<?xml version="1.0" encoding="utf-8"?>
+<xs:schema targetNamespace="http://www.Dummy.com/Common/Types"
+  xmlns:xs="http://www.w3.org/2001/XMLSchema" xmlns="http://www.Dummy.com/Common/Types"
+  >
+  <xs:import namespace="http://www.Dummy.com/Name/Types" schemaLocation="B.xsd" />
```

Comment:
This is correct `file.wsdl` imports `A.xsd` and `B.xsd`, `A.xsd` imports `B.xsd` and `B.xsd` imports `A.xsd`

With the code currently published `file.wsdl` would import `A.xsd` which would then import `B.xsd` and then `A.xsd` and so on

With the change `file.xsdl` would import `A.xsd` which would then import `B.xsd`, which would then use the reference to the previously imported `B.xsd`

If `A.xsd` referenced `A.xsd` and `B.xsd` referenced `B.xsd` it would work the exact same.